### PR TITLE
script: migrate keepers from q3 to q4 drippers

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**
solution to #878

### CODE REVIEW

**Review by petrovska-petro — APPROVED** [review]
migrations looks good:

- cancels upkeeps 127&128 which are from last quarter ✅ 

- retrieve link balances from those cancel ✅ 

- sets gas limit to 300k as prev jobs ✅ 

- register new drippers and top them up with link balance ✅

File: scripts/issue/878/migrate_drippers_q3_q4.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    SAFE.chainlink.register_upkeep(
+        "RemBadgerDripper2022Q4", r.drippers.rembadger_2022_q4, 300_000, LINK_MANTISSA
+    )
+
+    # update watchlist on the gas station
+    SAFE.badger.set_gas_station_watchlist()
+
+    if not sim:
+        SAFE.post_safe_tx(call_trace=True)
+    else:
```
Comment (sajanrajdev) [inline scripts/issue/878/migrate_drippers_q3_q4.py:59]: Just out of curiosity, why do we include a posting when simulating?

Comment (sajanrajdev) [reply scripts/issue/878/migrate_drippers_q3_q4.py:59]: Nvm, I guess it is so that the snapshot print, got it.

Comment (gosuto-inzasheru) [reply scripts/issue/878/migrate_drippers_q3_q4.py:59]: well and sometimes it is good to have `.preview` run and see it run successfully.

**Review by sajanrajdev — APPROVED** [review]
- Cancels previous upkeeps and withdraws their outstanding LINK balance
- Registers new upkeeps with the same gas limit as the previous ones and tops them up with 50 LINK
- Simulation runs successfully:
```
snapshot result for 0x86cbD0ce0c087b482782c181dA8d191De18C8275:
               balance_before          balance_after          balance_delta
symbol
LINK    64.326337510456059545  89.878640199184261734  25.552302688728202189
```

LINK balance sheet LG: 62.8 + 62.7 + 64.3 + (50) + (50) = 89.8

Given that only ~12 LINK were spent for the previous quarter, reducing the amounts to use should be good.

LGTM

**Pull request merged**